yasa/hypno.py

```diff
@@ -280,6 +300,7 @@ def __init__(self, values, n_stages=5, *, freq="30s", start=None, scorer=None):
         self._labels = labels
         self._mapping = mapping
         self._scorer = scorer
+        self._proba = proba
 
     def __repr__(self):
```

Comment:
I'm not sure what causes this, but note the differences in iPython:

```
In: pd.read_csv
Out: <function pandas.io.parsers.readers.read_csv(filepath_or_buffer: 'FilePath ...

In: h.as_int
Out:
<bound method Hypnogram.as_int of <Hypnogram | 3 epochs x 30s (1.50 minutes), 3 stages>
 - Use `.hypno` to get the string values as a pandas.Series
 - Use `.as_int()` to get the integer values as a pandas.Series
 - Use `.plot_hypnogram()` to plot the hypnogram
See the online documentation for more details.>
```

I often hit enter on methods without calling them, just to get a quick (if ugly) view of the arguments and such. When I do this on the Hypnogram methods it just returns the object string representation. I'm not sure what is necessary to switch it over.

Reply:
I think you just need to add a "?" at the end 👇 

<img width="691" alt="image" src="https://github.com/user-attachments/assets/c0138d4a-e72e-4814-988d-453d723a1f26" />

Reply:
I had no idea, thanks!!